examples/nats-basic-pub/nats-basic-pub.md

```diff
@@ -1,6 +1,6 @@
 # NATS client - Publish message
 
-NATS implements a publish-subscribe message distribution model. A publisher sends a message to a subject and any active subscriber listening to that subject can consume the message. In this example, the NATS client is used to produce a message to a subject.
+The NATS client allows publishing messages to a given subject. A NATS client is created by passing the URL of the NATS broker. To publish messages, the `publishMessage` method is used which requires the message and subject as arguments. Use it when you want to publish messages that can be received by one or more subscribers.
```

Comment:
Shall we reword the `Use it when you want to publish messages that can be received by one or more subscribers.` to `Use it to publish messages that can be received by one or more subscribers.` since our convention is not to address the user directly? 
@shafreenAnfar WDYT?